wireapp/kalium
# feat(mls-conversation): lower update keying materials threshold for debug builds (AR-2163)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
Make updateKeyingMaterialsThreshold lower for debug builds to make it easier to test.
### Issues

The reals threshold is 90 days! and that makes it hard to test by the QA, but by this change it would be 1 minute for debug builds.

### Causes (Optional)

Help the QA to test MLS Conversations behaviours.

### Solutions

Lower the value for debug builds.

### Dependencies (Optional)

Release with AR PR.

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Create an MLS conversation, then go to the conversation details, remember the keying material update value, close the app and open it after one minute. after the sync check the group details again the keying materials update value should be changed.

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
